### ISSUE TITLE
LINQPad 8 compatibility

### ISSUE THREAD

**Issue by Husqvik** [lethek/Serilog.Sinks.LINQPad#7]
It seems that LINQPad API changed a bit with version 8, so the sink no longer works.

```
Caught exception while emitting to sink Serilog.Sinks.LINQPad.LINQPadSink: System.MissingMethodException: Method not found: 'System.Object LINQPad.Util.RawHtml(System.String)'.
   at Serilog.Sinks.LINQPad.LINQPadSink.Emit(LogEvent logEvent)
   at Serilog.Core.Sinks.SafeAggregateSink.Emit(LogEvent logEvent)
```

**Comment by lethek** [comment]
Thanks @Husqvik, I had no idea LINQPad 8 was even available yet!

It looks like the official LINQPad.Runtime nuget package that the sink relied on to interact with LINQPad has not yet been updated for v8 either.

My solution is to reflect the necessary APIs instead upon initializing the sink. I've made the necessary changes and published a new [version 4.0.0 of the nuget package](https://www.nuget.org/packages/Serilog.Sinks.LINQPad/4.0.0). If you don't mind giving that a try and letting me know how you go, that'd be great.

As a bonus that also means the sink no longer has that 26MB dependency on LINQPad.Runtime either.

**Comment by Husqvik** [comment]
Thanks.

Tested both LINQPad 7 and 8 and works fine in both.

I like the solution. For these particular cases I find reflection as good option as you can simply manage many dependency versions of runtime loaded assembly (that you don't control) at once.
